### DISCUSSION TITLE
What is the output quality.

### DISCUSSION THREAD

**Discussion by max5354** [z-huang/music#118] [General]
What is the output quality of this app? Means if we play songs that what is output bitrate of song?

**Comment by z-huang** [comment]
It depends on what quality YouTube gives us. It seems that the highest is 128kbps.

**Reply by max5354** [reply]
> It depends on what quality YouTube gives us. It seems that the highest is 128kbps.

You said that you will add option to change audio quality like low, medium, high. So what is the actual quality of songs if we select "high" quality?

**Reply by z-huang** [reply]
The highest possible.

**Reply by max5354** [reply]
> The highest possible.

Ohk...got it...waiting for "audio quality" option.
